packages/nuxt/package.json

```
@@ -0,0 +1,76 @@
+{
+  "name": "@asgardeo/nuxt",
+  "version": "0.0.1",
+  "description": "Nuxt module for Asgardeo - Authentication and Identity Management",
+  "author": "WSO2",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/asgardeo/web-ui-sdks/tree/main/packages/nuxt#readme",
+  "bugs": {
+    "url": "https://github.com/asgardeo/web-ui-sdks/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/asgardeo/web-ui-sdks",
+    "directory": "packages/nuxt"
+  },
+  "keywords": [
+    "asgardeo",
+    "authentication",
+    "identity",
+    "oauth",
+    "oidc",
+    "nuxt",
+    "nuxt3",
+    "login",
+    "sso",
+    "identity-management"
+  ],
+  "type": "module",
+  "exports": {
+    ".": {
+      "types": "./dist/types.d.mts",
```

Is `m`(ts | js) expected here?
```suggestion
      "types": "./dist/types.d.mts",
```